Content.Server/Mobs/CritMobActionsSystem.cs

```diff
@@ -47,6 +52,12 @@ private void OnFakeDeath(EntityUid uid, MobStateActionsComponent component, Crit
         if (!_mobState.IsCritical(uid))
             return;
 
+        if (HasComp<MutedComponent>(uid))
```

Comment:
Please replace this with a call to ActionBlockerSystem's `CanSpeak()` method.

Reply:
That won't work in this case because crit mobs can't speak